test/iterable_extensions_test.dart

```diff
@@ -116,4 +116,34 @@ void main() {
     var coll = [1, 2, 3];
     expect(await coll.map(Future.value).awaitAll(), coll);
   });
+
+  group('isEmpty', () {
```

Comment:
need tests for FIrstOrNull

Reply:
Hmm. Well I didn't _add_ it now so it _should_ already have tests. I'll add some